src/asm/macro.c

```diff
@@ -168,12 +168,17 @@ void macro_ShiftCurrentArgs(int32_t count)
 {
 	if (!macroArgs) {
 		error("Cannot shift macro arguments outside of a macro\n");
-	} else if (macroArgs->shift < macroArgs->nbArgs) {
+	} else if (count > 0 && (count > macroArgs->nbArgs ||
```

Comment:
Just a note, nothing serious; but our code style is inconsistent with this. I put the `||` on the following line, aligning the expressions.

I'm fine if you want to keep it like this, but we really need #612, lol.

Reply:
Your change is fine. I assumed it was okay since checkpatch didn't complain.

Reply:
`checkpatch` doesn't fully cover the code style.

Also, since it includes some sanity checks (e.g. control flow in macros, etc.), I'd still keep it after we create a `.clang-format`.